pages/common/ngs.md

```diff
@@ -0,0 +1,16 @@
+# ngs
+
+> Fully featured scripting language created specifically for Ops. REPL is being developed.
+> More information: <https://github.com/ngs-lang/ngs>.
+
+- Execute a command:
```

Comment:
It isn't a command, it's NGS code, or a expression, because you can't use a Bash-like `echo` command, you should run `ngs -e 'echo("hello")'`
```suggestion
- Execute a NGS code:
```